.circleci/config.yml

```diff
@@ -59,22 +59,22 @@ jobs:
                 name: Run pylint
                 command: |
                     . venv/bin/activate
-                    pylint tests/ dash_bio/utils/ dash_bio/component_factory/
+                    pylint tests/ dash_bio/utils/ dash_bio/component_factory/ --ignore tests/test_skeleton.py
                 when: always
 
             - run:
                 name: Run flake8
                 command: |
                     . venv/bin/activate
-                    flake8 --max-line-length=100 tests/ dash_bio/utils/ dash_bio/component_factory/
+                    flake8 --max-line-length=100 tests/ dash_bio/utils/ dash_bio/component_factory/ --ignore tests/test_skeleton.py
```

Comment:
You don't want to make this change either: With `flake8`, the `--ignore` option is for specifying error codes, not files.
Reference: http://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-ignore